huxley/www/js/components/AdvisorFeedbackView.js

```diff
@@ -0,0 +1,191 @@
+      var schoolID = CurrentUserStore.getCurrentUser().school.id;
+      var delegates = DelegateStore.getSchoolDelegates(schoolID);
+      var feedback = this.prepareFeedback(delegates, this.state.assignments);
+      this.setState({
+        delegates: delegates,
+        feedback: feedback
+      });
+    });
+  },
+
+  componentWillUnmount: function() {
+    this._committeesToken && this._committeesToken.remove();
+    this._countriesToken && this._countriesToken.remove();
+    this._delegatesToken && this._delegatesToken.remove();
+    this._assignmentsToken && this._assignmentsToken.remove();
+  },
+
+  render: function() {
+    return (
+      <InnerView>
+        <h2>Feedback</h2>
+        <p>
+          Here you can view chairs feedback on your delegates, as well as their attendance.
+        </p>
+          <div className="table-container">
+            <table className="table highlight-cells">
+              <thead>
+                <tr>
+                  <th>Committee</th>
+                  <th>Country</th>
+                  <th>1</th>
+                  <th>2</th>
+                  <th>3</th>
+                  <th>4</th>
+                  <th>Summary</th>
+                </tr>
+              </thead>
+              <tbody>
+                {this.renderAssignmentRows()}
+              </tbody>
+            </table>
+          </div>
+      </InnerView>
+    );
+  },
+
+  renderAssignmentRows: function() {
+    var assignments = this.state.assignments
+    var committees = this.state.committees;
+    var countries = this.state.countries;
+    var feedback = this.state.feedback;
+    return assignments.map(function(assignment) {
+      var delegates = feedback[assignment.id];
+      if (delegates == null) {
+        return;
+      }
+      return (
+        <tr>
+          <td>{committees[assignment.committee].name}</td>
+          <td>{countries[assignment.country].name}
+          </td>
+          <td>
+            <input
+              className="choice"
+              type="checkbox"
+              checked={delegates.session_one}
+              disabled
+            />
+          </td>
+          <td>
+            <input
+              className="choice"
+              type="checkbox"
+              checked={delegates.session_two}
+              disabled
+            />
+          </td>
+          <td>
+            <input
+              className="choice"
+              type="checkbox"
+              checked={delegates.session_three}
+              disabled
+            />
+          </td>
+          <td>
+            <input
+              className="choice"
+              type="checkbox"
+              checked={delegates.session_four}
+              disabled
+            />
+          </td>
+          <textarea
+              className="text-input"
+              style={{"width": "95%"}}
+              defaultValue={delegates.published_summary}
+              disabled
+            />
```

Comment:
This closing tag, and the attributes in this text area need to be unindented one space